frontend/src/lib/components/LemonInput/LemonNumericInput.tsx

```diff
@@ -0,0 +1,116 @@
+import './LemonInput.scss'
+import React, { useRef, useState } from 'react'
+import { LemonRow, LemonRowProps } from 'lib/components/LemonRow'
+import clsx from 'clsx'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconClose } from 'lib/components/icons'
+
+export interface LemonNumericInputProps
```

Comment:
Can we just intelligently work with `type="number"` in `LemonInput`? This would cut down on code duplication and would make for a more intuitive developer experience.